op-chain-ops/genesis/config.go

```diff
@@ -738,10 +738,10 @@ func (d *L1Deployments) Check(deployConfig *DeployConfig) error {
 	}
 	for i := 0; i < val.NumField(); i++ {
 		name := val.Type().Field(i).Name
-		// Skip the non production ready contracts
-		if name == "DisputeGameFactory" ||
-			name == "DisputeGameFactoryProxy" ||
-			name == "BlockOracle" {
+		if !deployConfig.UseFaultProofs &&
+			(name == "DisputeGameFactory" ||
+				name == "DisputeGameFactoryProxy" ||
+				name == "BlockOracle") {
```

Comment:
We don't have a `BlockOracle` anymore (I don't think - pretty sure it was part of FPA).